lib/shared.js

```diff
@@ -165,6 +159,17 @@ module.exports = {
 
 				// prefer called with
 				'jest/prefer-called-with': 'error',
+			},
+		},
+
+		{
+			files: ['**/test/**', '**/__tests__/**/*.ts?(x)', '**/?(*.)+(spec|test).ts?(x)'],
```

Comment:
Should we make the `**/test/**` variant consistent with the `**/__tests__/**` variant and include the extensions?

Reply:
Ah, good point.